### PR TITLE
Added CardView and RecyclerView for workouts

### PR DESCRIPTION
**Pull request by pranitishettiwar**
TODOs:
- Button click for Create New workout activity
- Change workout detail activity to dialogFragment
- Add details on pre-defined workout detail activity

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,7 +3,7 @@
 /local.properties
 /.idea/workspace.xml
 /.idea/libraries
-/.idea/*
+.idea/*
```
Comment (statham) [inline .gitignore:6]: sweet!

File: app/src/main/java/com/kyphipraniti/fitnesstasks/adapters/RVAdapter.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import com.kyphipraniti.fitnesstasks.activities.WorkoutDetailActivity;
+
+/**
+ * Created by praniti on 10/11/17.
+ */
+
+public class RVAdapter extends RecyclerView.Adapter<RVAdapter.MyViewHolder> {
+
+    ArrayList<String> workoutNames;
+    ArrayList<Integer> workoutImages;
```
Comment (statham) [inline app/src/main/java/com/kyphipraniti/fitnesstasks/adapters/RVAdapter.java]: these two array lists should probably be a single array list of `Task` objects yeah?

File: app/src/main/java/com/kyphipraniti/fitnesstasks/adapters/RVAdapter.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            }
+        });
+    }
+
+    @Override
+    public int getItemCount() {
+        return workoutNames.size();
+    }
+
+    public class MyViewHolder extends RecyclerView.ViewHolder {
```
Comment (statham) [inline app/src/main/java/com/kyphipraniti/fitnesstasks/adapters/RVAdapter.java]: could you rename this to just ViewHolder

**Review by statham — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/kyphipraniti/fitnesstasks/adapters/RVAdapter.java (hunk 2 above)
Comment (pranitishettiwar) [reply app/src/main/java/com/kyphipraniti/fitnesstasks/adapters/RVAdapter.java]: We can keep this for now, and change it when we integrate.

File: app/src/main/java/com/kyphipraniti/fitnesstasks/adapters/RVAdapter.java (hunk 3 above)
Comment (pranitishettiwar) [reply app/src/main/java/com/kyphipraniti/fitnesstasks/adapters/RVAdapter.java]: done

**Pull request merged**